### ISSUE TITLE
Protractor, WebDriver chain maxes out at ChromeDriver 2.46 not compatible with Chrome 73

### ISSUE THREAD

**Issue by yogeshgadge** [angular/angular-cli#14025]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ X] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

Not related.

### Description

Protractor version 5.4.2 leads to max ChromeDriver 2.46 
While the latest Chrome version is 74 (minus 1 is 73) they are not compatible with ChromeDriver version 2.46. (side point Canary is exception).

Not sure if the bug belongs to angular-cli but our version 7.x.x maxes out at ChromeDriver 2.46 and we have latest Chrome browser version on our machines/test servers which leads into some incompatibility issues.

Bigger issue is Protractor's next version is 6 and that has breaking changes i.e. no control flow and must use async-await.

I am already using async-await with disabled  `SELENIUM_PROMISE_MANAGER: false`  it seems the ChromeDriver 2.46 incompatibility with Chrome 73 may have made my e2e tests flaky. Especially that `browser.wait(ExpectedConditions.xxxxxxxx)` don't seem to work.

TL;DR;

Chrome jumped the gun ditching protractor, webdriver-manager, CLI.
TBF protractor has 6.0 and webdriver-manager has 13.0 which is probably ready for Chrome 73 and above however Angular CLI is at 7.3.7 (and even the 8.beta) versions give us last protractor version of 5.4.2

https://github.com/angular/angular-cli/blob/4dd6d3357d57e07ccd7243ce7be953f5aacb39de/packages/schematics/angular/workspace/files/package.json.template#L42

## 🌍 Your Environment
<pre><code>
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 7.2.3
Node: 11.6.0
OS: win32 x64
Angular: 7.2.2
... animations, cdk, common, compiler, compiler-cli, core, forms
... http, language-service, material, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.12.3
@angular-devkit/build-angular     0.12.3
@angular-devkit/build-optimizer   0.12.3
@angular-devkit/build-webpack     0.12.3
@angular-devkit/core              7.2.3
@angular-devkit/schematics        7.2.3
@angular/cli                      7.2.3
@angular/flex-layout              7.0.0-beta.23
@ngtools/webpack                  7.2.3
@schematics/angular               7.2.3
@schematics/update                0.12.3
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.28.4

</code></pre>

**Anything else relevant?**
Related to Chrome versions 73+

**Comment by alan-agius4** [comment]
Hi, I think this issue should be redirected to protractor https://github.com/angular/protractor
